### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by pavolloffay**
Import from the old repository with initial implementation.

* tracing servlet filter
* integration tests
* mvnw with release configuration

### CODE REVIEW

**Comment by pavolloffay** [comment]
@yurishkuro @bensigelman could you please review?

@sjoerdtalsma could you please look at maven release config?

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  </dependencyManagement>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+
+      <distributionManagement>
+        <repository>
+          <id>bintray</id>
+          <url>https://api.bintray.com/maven/opentracing/maven/opentracing-java/;publish=1</url>
```
Comment (sjoerdtalsma) [inline pom.xml]: Same comment as in https://github.com/opentracing-contrib/java-jaxrs/pull/5,
shouldn't `opentracing-java` be changed to `java-web-servlet-filter` in this case?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - ./travis/publish.sh
+
+branches:
+  except:
+    - /^[0-9]/
+
+env:
+  global:
+  # Ex. travis encrypt GH_TOKEN=XXX-https://github.com/settings/tokens-XXX -r org/repo
+  - secure: "lhiFuGsY0MA6zs5bVcDBeseE5XB6Q2ioMfOyNos3m6rHcZpJojGhUWfg939veGprsDxeQozjLk2/Oeg6CwAZED81reSGqh+NfJJjOvWGThM9I439FkX5SqwDwEPJzezvgmtT+nh/kqh6PQ0dSx3OUdlGiEa3J6r1s7AlPA5FmFuxLkjqFcqKlUqbuxKdzWVh3yCUk+Rz70POhT6PFUYzwg3EOZt0pKfggkxv6PfbT9SxsyUjol0JzyYn9T+2xiA3V1GDoxncL9tjtm7pdMvzEGRjUWGqaaaGhD4oWQP2cfzLgbcakP8kHo/nOHTkibACJJXp/skyPe+QsicT5+yAJyp7U20bgrGLDPav4CUoHCGeGMz3eSgo3q5aaQTxHW+4S0PIRTRNiGaowIYdqhewJPsMGqEzF80/rF5dl81hDur71D/aSbL2WPBCHaGNPCTr69jZIhe4M4RGYbH4eq499xneP6J42ZYkPPzQxiZy5iyEo2qld703hb5fwHmeSRklZFODhLTUJC/41b+ucQ2b9B3t0f0LeamCzzB420wWWXSTqE8spG/mhgL25iNHGhdZT5o8eK2BXNFJRCsuvRXJ9dtA1nisDrp51vkLrWJa2bDwgQuqptJuJmYiBMScvodMcGpWrwVFDGwS/mBsxt/7ipQ4gaj1Wv2zFEG1I0iVwyk="
```
Comment (bhs) [inline .travis.yml:35]: @pavolloffay just confirming that you hand-generated these...

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+[![Travis](https://travis-ci.org/opentracing-contrib/java-web-servlet-filter.svg?branch=master)](https://travis-ci.org/opentracing-contrib/java-web-servlet-filter)
+
+This library provides instrumentation for Java Web Servlet applications.
+
+## Maven
+```xml
+   <dependency>
+     <groupId>io.opentracing.contrib</groupId>
+     <artifactId>opentracing-web-servlet-filter</artifactId>
+     <version>0.0.1.SNAPSHOT</version>
```
Comment (bhs) [inline README.md]: these are hard to keep up-to-date manually... maybe remove the version from the README? Or link to the sonatype or bintray search for this artifact to reveal the current released version?

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+   servletContext.setAttribute(TracingFilter.TRACER, tracer);
+   servletContext.setAttribute(TracingFilter.SPAN_DECORATORS, decorators); // optional, if no present SpanDecorator
+   .STANDARD_TAGS is applied
+   servletContext.setAttribute(TracingFilter.TRACING_DECISION, tracingDecision); // optional
+```
+
+## Accessing Server Span
+Current server span is accessible in HttpServletRequest attributes.
+```java
+   Span span = (Span)httpservletRequest.getAttribute(TracingFilter.ACTIVE_SPAN);
```
Comment (bhs) [inline README.md:26]: I guess you could create a static method on `TracingFilter` that takes a servlet request and encapsulates the cast... YMMV, no strong feelings.

**Review by bhs — APPROVED** [review]
Thanks!

File: README.md (hunk 3 above)
Comment (pavolloffay) [reply README.md]: I had the same feeling when I was writing it...

File: .travis.yml (hunk 2 above)
Comment (pavolloffay) [reply .travis.yml:35]: yes I have a nice shell script for it :)

File: README.md (hunk 4 above)
Comment (pavolloffay) [reply README.md:26]: I would like to make people aware that active span is in request attributes. Then they can get span in various ways depending what other framework are using. Accessing active server span will probably change with `java-activespan` repo.

**Pull request merged**